### DISCUSSION TITLE
Integration with NeuralForecast

### DISCUSSION THREAD

**Discussion by frknayk** [amazon-science/chronos-forecasting#179] [Ideas]
Hello everyone,

With using Nixtla's `neuralforecast` library, it is easy to fine-tune and comparing existing neural network based time-series foreacsting models. Do you plan to integrate Chronos to it ?

**Comment by lostella** [comment]
@frknayk thanks for the suggestion. Integrating the model into popular libraries is a good idea, but we do not have plans for that. I see there is an open issue about this https://github.com/Nixtla/neuralforecast/issues/1121.

**Reply by frknayk** [reply]
Thank you for reply @lostella, Is it allowed to integrate Chronos into another library like Neuralforecast or any other library for your side
